doc/api/fs.md

```diff
@@ -74,7 +74,7 @@ Most fs functions let you omit the callback argument. If you do, a default
 callback is used that rethrows errors. To get a trace to the original call
 site, set the `NODE_DEBUG` environment variable:
 
-```
+```txt
```

Comment:
This is how our doc tooling renders it:
![screenshot_20160716_163656](https://cloud.githubusercontent.com/assets/291301/16895116/90996f2e-4b73-11e6-8ac3-a161d1810958.png)

That doesn't look good enough to me.